File: HW9/hw9.java
public class hw9 {
    
    // adder method that accepts a variable number of arguments
    public static int adder(int... numbers) {
        int sum = 0; // Initialize sum to 0
        for (int num: numbers) {
          sum += num;
        }
        return sum;
    }

    public static void main(String[] args) {
        // Test cases with different numbers of arguments
        System.out.println("Sum 1 is " + adder(1, 2, 3, 4, 5, 6)); // Sum: 21
        System.out.println("Sum 2 is " + adder(1, 5));              // Sum: 6
        System.out.println("Sum 3 is " + adder());                  // Sum: 0
    }
}

File: HW9/hw9.cs
using System;

namespace hw9 {
	class hw9 {
		public static int Adder(params int[] numbers)
		{
			int sum = 0;
			
			foreach (int num in numbers) {
				sum += num;
			}
			
			return sum;
		}
		
		public static int Main(String [] args)
		{
			Console.WriteLine("Sum 1 is {0}", Adder(1,2,3,4,5,6));
			Console.WriteLine("Sum 2 is {0}", Adder(1,5));
			Console.WriteLine("Sum 3 is {0}", Adder());
			
			return 0;
		}
	}
}
